### PR TITLE
[WIP] More-actions button.

### PR DESCRIPTION
**Pull request by Ivansss**
@ChristophWurst @jancborchardt

### CODE REVIEW

File: js/rooms.js [hunk 1]
```diff
@@ -45,7 +45,14 @@ $(document).ready(function() {
 				success: function(data) {
 					$('#app-navigation ul').html('');
 					data.forEach(function(element) {
-						$('#app-navigation ul').append('<li><a href="#'+escapeHTML(element['id'])+'">'+escapeHTML(element['name'])+' <span class="utils">' + escapeHTML(element['count']) + '</span></a></li>');
+						$('#app-navigation ul').append(
```
Comment (ChristophWurst) [inline js/rooms.js]: there's an open PR https://github.com/nextcloud/spreed/pull/60 to replace this manual string concatenation with handlebars templating. This means the two PRs are conflicting 😟

Comment (jancborchardt) [inline js/rooms.js]: @ChristophWurst so I assume we should get in #60 first? We also need to coordinate with @LukasReschke’s #55 person selector.

Comment (LukasReschke) [inline js/rooms.js]: Just don't touch the PHP for now and we won't merge-conflict too hard… ;-)

**Comment by Ivansss** [comment]
@ChristophWurst When your branch is merged. How do we proceed here?

File: js/rooms.js [hunk 2]
```diff
@@ -45,7 +45,33 @@ $(document).ready(function() {
 				success: function(data) {
 					$('#app-navigation ul').html('');
 					data.forEach(function(element) {
-						$('#app-navigation ul').append('<li><a href="#'+escapeHTML(element['id'])+'">'+escapeHTML(element['name'])+' <span class="utils">' + escapeHTML(element['count']) + '</span></a></li>');
+						$('#spreedme-room-list').append(
+							'<li id="'+escapeHTML(element['name'])+'">'+
```
Comment (ChristophWurst) [inline js/rooms.js]: this code should be applied to here https://github.com/nextcloud/spreed/blob/e76242bee99b2a710e0b95e5e70ef59abf70553f/js/views/roomlistview.js#L29

File: js/rooms.js [hunk 3]
```diff
@@ -78,4 +104,18 @@ $(document).ready(function() {
 		}
 	};
 
+	$(document).click(function(e) {
```
Comment (ChristophWurst) [inline js/rooms.js]: I'm not a fan of those global handlers but it *should* work as a first version :-)

(Note: I've implemented such a menu for Mail too, maybe we can re-use some code)

Comment (jancborchardt) [inline js/rooms.js]: Yes we should – the menu is also implemented in Calendar (@georgehrke @raghunayyar) and News (@bernhardposselt) ;)

**Comment by jancborchardt** [comment]
@Ivansss remember to delete branches after merge ;)

**Pull request merged**